### PR TITLE
attempt to fix attrs

### PR DESCRIPTION
**Pull request by Lazyuki**
Another attempt at fixing #292 

It kind of works with `attrs` but I need to see why it's changing so many irrelevant tests.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -10,15 +10,15 @@ assignees: ""
 A clear and concise description of what the bug is.
 If you're planning on posting a screenshot, please paste the accompanying code here.
 
-**Screenshot**
+**Screenshot**
```
Comment (jasonwilliams) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: Where are these spaces coming from? Do you have prettier installed? Do I need to add some rules?

File: src/tests/suite/colorize-results/string-tagname_js.json [hunk 2]
```diff
@@ -109,7 +109,7 @@
 	},
 	{
 		"c": "div",
-		"t": "source.js meta.var.expr.js string.quoted.double.ts",
+		"t": "source.js meta.var.expr.js string.quoted.double.js.jsx",
```
Comment (jasonwilliams) [inline src/tests/suite/colorize-results/string-tagname_js.json:112]: How come the extensions have changed to jsx.js here?

Comment (Lazyuki) [reply src/tests/suite/colorize-results/string-tagname_js.json:112]: Yeah that's the part I'm trying to fix if possible. `source.ts#expression` doesn't parse JSX so I added `source.js.jsx` as one of the `includes`. But that changed some of these scopes in unwanted places. Although if you open VSCode you don't really see the difference in colors.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (Lazyuki) [reply .github/ISSUE_TEMPLATE/bug_report.md:13]: I assume prettier changed it. Not sure what rule it is though

**Comment by jasonwilliams** [comment]
> It kind of works with attrs but I need to see why it's changing so many irrelevant tests.

So it's because the [new rule](https://github.com/styled-components/vscode-styled-components/pull/394/files#diff-b101ec1f87c72617c31ad55ada58396a0295ac40c27883c0b95aee5ab773bde5R240) you've created is matching existing lines and that's why it's changing irrelevant tests. 
Here is an example with component.js

Here is your rule put into regex101 and a component fixture added as the body.
https://regex101.com/r/GcqXnV/1

Is there a way we can update an existing rule without having a very similar one? Or we will need to change this rule so it doesn't affect existing lines.

Was there a reason for you making a new rule? can this [this one](https://github.com/styled-components/vscode-styled-components/blob/main/syntaxes/styled-components.json#L7) not be updated instead? This is the rule your is conflicting with.

File: syntaxes/styled-components.json [hunk 3]
```diff
@@ -75,7 +75,7 @@
     },
     {
       "contentName": "source.css.scss",
-      "begin": "(?:}>|\\)\\))(`)",
+      "begin": "(?:}>)(`)",
```
Comment (jasonwilliams) [inline syntaxes/styled-components.json:78]: `(?:}>|\\)\\))(`)` was needed to catch multi-line rules. 
[here](https://regex101.com/r/NBlmzn/1) is an example

**Pull request closed without merging**